src/components/liquality/ConfirmLiqualityTransaction.vue

```diff
@@ -203,23 +216,46 @@ export default class ConfirmLiqualityTransaction extends Vue {
 
   environmentContext = EnvironmentContextProviderService.getEnvironmentContext();
 
+  // eslint-disable-next-line class-methods-use-this
+  @Emit('successConfirmation')
+  errorOnConnection() {
+    this.confirmTxState.send('error');
+    const error = new LiqualityError();
+    return [error.message, this.txId, error.urlToMoreInformation,
+      error.errorType, error.messageToUserOnLink];
+  }
```

Comment:
I don't get this, you are instantiating an empty LiqualityError and returning its part... is this truly needed?